### ISSUE TITLE
Issue in unicode/UTF-8 handling

### ISSUE THREAD

**Issue by kingwatam** [mdlincoln/clipr#57]
I've been using clipr::write_clip() for a while. It's been very helpful in transfering data from a data frame to Excel, for example. 

However, clipr::write_clip() is unable to copy data in UTF-8/unicode. Changing the locale using Sys.setlocale() doesn't help, other than turning the unicode text into gibberish when set to the corresponding locale, and the text is copied as unicode when set to English.  The console behaves in the opposite manner - it displays the correct foreign text when Sys.setlocale() is set to the correct locale, whereas it displays unicode when set to English. 

Other functions such as write.xlsx2() work as expected. I have not found a way to make write_clip() work in this situation. Suggestions are most welcome in terms of how to convert the data frame before using write_clip. writeClipboard(format = 13) also works, but formatting is all messed up for tables.

**Comment by mdlincoln** [comment]
Thank you for your patience, I realize it's been some time since you originally posted this issue.

Re: #63 I am curious if this issue persists using R 4.2 on windows. Could you try using the latest version of clipr under R 4.2 and report your results?

to install:

```r
# install.packages("remotes")
remotes::install_github("mdlincoln/clipr", ref="b4d69e3c06a66566d64096e2784246041a1d16b0")
```

And please post the results you get, along with the output of `devtools::session_info()`

**Comment by kingwatam** [comment]
> Thank you for your patience, I realize it's been some time since you originally posted this issue.
> 
> Re: #63 I am curious if this issue persists using R 4.2 on windows. Could you try using the latest version of clipr under R 4.2 and report your results?
> 
> to install:
> 
> ```r
> # install.packages("remotes")
> remotes::install_github("mdlincoln/clipr", ref="b4d69e3c06a66566d64096e2784246041a1d16b0")
> ```
> 
> And please post the results you get, along with the output of `devtools::session_info()`

4.2 isn't officially out yet. I can do that when it comes out (if only updating/downgrading R isn't so much of a hassle).

**Comment by kingwatam** [comment]
I'm happy to report that this issue has now been fixed (using R 4.2). Thank you.
